templates/plugin/container_gateway.yml.erb

```
@@ -3,3 +3,4 @@
 :enabled: <%= @module_enabled %>
 :pulp_endpoint: <%= scope.lookupvar("foreman_proxy::plugin::container_gateway::pulp_endpoint") %>
 :sqlite_db_path: <%= scope.lookupvar("foreman_proxy::plugin::container_gateway::sqlite_db_path") %>
+:sqlite_timeout: <%= scope.lookupvar("foreman_proxy::plugin::container_gateway::sqlite_timeout") %>
```

```suggestion
<% if scope.lookupvar("foreman_proxy::plugin::container_gateway::sqlite_timeout") -%>
:sqlite_timeout: <%= scope.lookupvar("foreman_proxy::plugin::container_gateway::sqlite_timeout") %>
<% end -%>
```

Otherwise you end up with an `:sqlite_timeout: ` line in the case you did not specify a timeout

---

ah nice, I was wondering about this